src/components/Footer.jsx

```diff
@@ -0,0 +1,272 @@
+  .footer-title-breaker {
+    display: inline;
+  }
+
+  .social-media {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    position: absolute;
+    right: 10%;
+    top: 43%;
+  }
+
+  .media-link {
+    margin: 0 20px;
+  }
+
+  .media-icon {
+    width: 56px;
+    height: 56px;
+  }
+
+  .further-info {
+    width: 80%;
+
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: flex-end;
+  }
+
+  .text-info {
+    width: 60%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: flex-end;
+  }
+
+  .company-name {
+    color: #000;
+    font-family: Calistoga;
+    font-size: 18px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 19px;
+    letter-spacing: 2.7px;
+    margin-bottom: 2.5rem;
+  }
+
+  .address-detail {
+    color: #000;
+    font-family: "AvenirMedium", sans-serif;
+    font-size: 18px;
+    font-weight: 500;
+    line-height: 30px; /* 166.667% */
+  }
+
+  address {
+    font-style: normal;
+  }
+
+  .company-email {
+    cursor: pointer;
+    font-weight: 900;
+  }
+
+  .info-links {
+    color: #000;
+    font-family: "AvenirMedium", sans-serif;
+    font-size: 18px;
+    font-style: normal;
+    font-weight: 500;
+  }
+
+  .link {
+    cursor: pointer;
+    list-style-type: none;
+
+    margin-top: 3px;
+  }
+
+  .language-select {
+    appearance: none;
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    cursor: pointer;
+
+    border-radius: 12px;
+    border: 1px solid #000;
+
+    width: 20%;
+    background: #fff;
+    display: flex;
+    margin-top: 8.5rem;
+    padding: 8px 24px;
+    justify-content: center;
+    align-items: center;
+
+    color: #000;
+
+    font-family: "AvenirMedium", sans-serif;
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: normal;
+  }
+
+  @media (max-width: 1200px) {
+    .social-media {
+      right: 8%;
+    }
+  }
+
+  @media (max-width: 985px) {
+    padding: 1.5rem 0.8rem 1rem;
+    .social-media {
+      right: 6%;
+    }
+  }
+
+  @media (max-width: 744px) {
+    position: none;
+
+    .footer-title {
+      margin-bottom: 0;
+      font-size: 1rem;
+      line-height: 19px;
+    }
+
+    .footer-title-breaker {
+      display: inline;
+    }
+
+    .social-media {
+      margin: 40px auto;
+      position: initial;
+      right: auto;
+      top: auto;
+    }
+
+    .media-icon {
+      width: 40px;
+      height: 40px;
+    }
+
+    .further-info {
+      width: 100%;
+      padding: 0 0.5rem;
+      justify-content: space-around;
+      align-items: center;
+    }
+
+    .text-info {
+      align-items: flex-end;
+    }
+
+    .company-name {
+      font-size: 19px;
+    }
+
+    .address-detail {
+      font-size: 1rem;
+    }
+
+    .info-links {
+      font-size: 1rem;
+    }
+
+    .language-select {
+      width: 30%;
+      margin-top: 1.5rem;
+    }
+  }
+

```

Comment:
![pr](https://github.com/Technigo/project-design-handoff-vite/assets/144723838/5668572c-538c-4603-93a6-7582a8195613)
In small devices it's all a bit smashed up :) and the "login it's wrapping" , the text in the button it's not centered